server/start.go

```diff
@@ -1,26 +1,26 @@
 package server
 
 import (
+	"context"
 	"fmt"
 	"net"
 	"os"
 	"runtime/pprof"
-	"time"
 
+	pruningtypes "cosmossdk.io/store/pruning/types"
 	"github.com/cometbft/cometbft/abci/server"
 	tcmd "github.com/cometbft/cometbft/cmd/cometbft/commands"
 	"github.com/cometbft/cometbft/node"
 	"github.com/cometbft/cometbft/p2p"
 	pvm "github.com/cometbft/cometbft/privval"
 	"github.com/cometbft/cometbft/proxy"
 	"github.com/cometbft/cometbft/rpc/client/local"
+	"github.com/neilotoole/errgroup"
```

Comment:
it seems can be replaced by builtin[sync/errgroup](https://pkg.go.dev/golang.org/x/sync/errgroup), according to the [readme](https://github.com/neilotoole/errgroup).